domains/_github-pages-challenge-hungthai1401.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hungthai1401",
+        "email": "hungthai1401.it@gmail.com"
+    },
+    "record": {
+        "TXT": "5e203b636d9f5f74655ec43b7cb2b1"
```

Comment:
Invalid GitHub Page file verification name. See docs for more information: https://docs.is-a.dev/guides/github-pages